packages/@aws-cdk-testing/cli-integ/lib/package-sources/repo-source.ts

```diff
@@ -96,7 +97,7 @@ async function findYarnPackages(root: string): Promise<Record<string, string>> {
  * Find the root directory of the repo from the current directory
  */
 export async function autoFindRoot() {
-  const found = await findUp('release.json');
+  const found = findUp('release.json');
```

Comment:
Odd that this was ever awaited. Looking in the `findUp` function, I am not seeing async calls, but is there any chance this await is necessary?

Reply:
TS compiler said it's not needed, so I removed it. It doesn't return a promise, so there's no need to await anything here.